### DISCUSSION TITLE
How To Access Admin Page ?

### DISCUSSION THREAD

**Discussion by taufiqpsumarna** [arifszn/ezfolio#9] [Q&A] [answered]
Thank you for the application. 
It's very Beautiful, simple but powerful for me to manage my portfolio website.

But How i can access the admin page ?

I can only access frontend with this url
http://<ip_address>/ezfolio/public/

**Comment by arifszn** [comment, marked as answer]
You can access it from `/admin`.

**Reply by taufiqpsumarna** [reply]
![image](https://user-images.githubusercontent.com/35789663/153752945-819eb5d0-6846-4187-9fdf-09552d19a560.png)

I tried that but failed

**Reply by taufiqpsumarna** [reply]
![image](https://user-images.githubusercontent.com/35789663/153753116-52efd679-0bb5-44bd-b7ab-35456c2813f2.png)
 I am using **npm run watch**

**Reply by arifszn** [reply]
http://<ip_address>/ezfolio/public/admin

**Reply by taufiqpsumarna** [reply]
![image](https://user-images.githubusercontent.com/35789663/153753725-63a33067-6dff-4a6c-a40a-0502bf24b7f1.png)


But i can't find admin page in public folder, I have cloned your repository

**Reply by arifszn** [reply]
Maybe cause of the public prefix. Follow '`My site url has /public. What should I do?` from https://arifszn.com/ezfolio/docs/faq. Or for local development, use laragon.

**Reply by taufiqpsumarna** [reply]
Great, it's work like a charm, i tried using virtualbox before than i using laragon and it's work very well 😁 thank you...
![image](https://user-images.githubusercontent.com/35789663/153754805-7a65e9ca-98b7-4e1d-b223-4752f008de00.png)
